R/data-combined.R

```diff
@@ -0,0 +1,628 @@
+#' @title Object combining simulated and observed data
+#'
+#' @description
+#'
+#' A class for storing simulated and/or observed in a single dataframe, which
+#' can be further used for visualization methods.
+#'
+#' Additionally, it allows:
+#'
+#' - Grouping different (simulated and/or observed) datasets with grouping
+#' labels.
+#'
+#' - Transforming data (with given offsets and scale factors).
+#'
+#' @param groups A string or a list of strings assigning the data set to a
+#'   group. If an entry within the list is `NULL`, the corresponding data set is
+#'   not assigned to any group (and the corresponding entry in the dataframe
+#'   will be an `NA`). If provided, `groups` must have the same length as
+#'   `dataSets` and/or `simulationResults`. If no grouping is specified for any
```

Comment:
This is not accurate. It is the same lenght as simulationResults$quantityPath

```
> length(simResults)
[1] 13
> myCombDat$addSimulationResults(simulationResults = simResults, groups = list("distal"))
 Error in validateIsOfLength(arg, expectedLength) : 
$: Object should be of length '5', but is of length '1' instead. myCombDat: Object should be of length '5', but is of length '1' instead. addSimulationResults: Object should be of length '5', but is of length '1' instead.
```

Reply:
I've changed the docs to reflect that.